tests/baselines/reference/witness.errors.txt

```diff
@@ -2,24 +2,17 @@ witness.ts(4,21): error TS2372: Parameter 'pInit' cannot reference itself.
 witness.ts(8,14): error TS2729: Property 'x' is used before its initialization.
 witness.ts(20,5): error TS2403: Subsequent variable declarations must have the same type.  Variable 'a' must be of type 'any', but here has type 'never'.
 witness.ts(28,12): error TS2695: Left side of comma operator is unused and has no side effects.
-witness.ts(29,5): error TS2403: Subsequent variable declarations must have the same type.  Variable 'co1' must be of type 'any', but here has type 'number'.

```

Comment:
it seems that this is even a small improvement since the intention of the caller was not to raise those errors when the type is the `errorType`, see: https://github.com/microsoft/TypeScript/blob/32b618c2d8f42948e7d6fa2e9b264079e30b5349/src/compiler/checker.ts#L42290-L42296

Reply:
Swinging back here, it seems sorta odd just from the PoV that a self referencing thing will just not have any errors. It seems like this test is mainly to "witness" the type of these expressions in errors, but maybe it's fine. Odd that the `.types` file does not show a change?

Reply:
Will have to do some forensics but I'm not sure why we even want to ignore these anyway...